app/(aux)/prayerCircleInfo.js

```diff
@@ -0,0 +1,565 @@
+								
+								<StyledView className='z-10 flex-row justify-evenly'>
+									<StyledView className='flex'>			
+										<StyledPressable
+											className='flex-col w-auto items-center justify-center'
+											onPress={() => (router.push('https://www.linkedin.com/in/nason-allen/'))}
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/nason.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Nason Allen
+											</StyledText>
+										</StyledPressable>							
+									</StyledView>
+
+									<StyledView className='flex'>		
+										<StyledPressable
+											className='flex-col w-auto items-center justify-center'
+											onPress={() => (router.push('https://www.linkedin.com/in/justin-ayres-25576921b/'))}
+										>							
+											<StyledImage
+												className='rounded-xl'
+												style={{ width: 60, height: 60 }}
+												source={require('../../assets/devs/justin.jpg')}
+											/>
+											<StyledText className={`font-[600] text-offwhite text-[20px]`}>
+												Justin Ayres
+											</StyledText>
+										</StyledPressable>							
+									</StyledView>
+								</StyledView>
+							
+								<StyledImage
+									className='z-0 absolute top-2 self-center justify-center w-[260px] h-[260px]'															
+									source={require('../../assets/spiral/spiral.png')}
+								/>
+
+							</StyledView>
+						</StyledView> */}
+
+						<View className='relative pb-[75px]'></View>
+					</StyledView>
+				</ScrollView>
+
+				<StyledGradient
+					pointerEvents='none'
+					start={{ x: 0, y: 0.1 }}
+					end={{ x: 0, y: 1 }}
+					style={{ height: 120 }}
+					className='absolute w-screen'
+					colors={['#121212ee', 'transparent']}
+				/>
+				<StyledView
+					style={{
+						top: Platform.OS == 'android' ? 15 : 15 // Need Alex to test this, it's perfect on Andriod
+					}}
+					className='absolute w-screen flex-row items-center justify-between px-[15px]'
+				>
+					<StyledText className='text-4xl font-bold text-offwhite'>
+						Prayer Circle
+					</StyledText>
+					<Button
+						icon='document-text'
+						iconColor={'#FFFBFC'}
+						iconSize={26}
+						width={'w-[65px]'}
+						height={'h-[35px]'}
+						bgColor={'bg-transparent'}
+						textColor={'text-offwhite'}
+						borderColor={'border-offwhite'}
+						btnStyles='border-2'
+						press={() =>
+							handleModalPress(
+								'tos',
+								['65%', '85%'],
+								'Terms of Service',
+								''
+							)
+						}
+					></Button>
+				</StyledView>
+				<StyledGradient
+					pointerEvents='none'
+					start={{ x: 0, y: 0.1 }}
+					end={{ x: 0, y: 1 }}
+					style={{ height: 100, bottom: 0 }}
+					className='absolute w-screen rotate-180'
+					colors={['#121212ee', 'transparent']}
+				/>
+
+				<StyledView
```

Comment:
The `bottom` style here needs to be dynamic. it looks way too high on iOS. check how circleSettings does this.

Reply:
The flow of the pages isn't the same, without your device I have no way to getting what you are looking for. @dev-arekusandoru please make a commit where you only implement an addition to the code allowing iOS device to have proper bottom styling. Thank you.